tensorflow/compiler/xla/service/gpu/gpu_fusible.cc

```diff
@@ -133,17 +133,18 @@ bool ShapesCompatibleForMultiOutputFusion(const HloInstruction& instr1,
 
 bool IsInputFusibleScatter(const HloInstruction& instr) {
   if (instr.opcode() == HloOpcode::kScatter ||
-     (instr.opcode() == HloOpcode::kFusion &&
-      instr.fusion_kind() == HloInstruction::FusionKind::kInput &&
-      instr.fused_expression_root()->opcode() == HloOpcode::kScatter)) {
+      (instr.opcode() == HloOpcode::kFusion &&
+       instr.fusion_kind() == HloInstruction::FusionKind::kInput &&
```

Comment:
is this your editor adding spaces?

Reply:
I added the spaces to comply with the coding style as referenced here: https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#c-coding-style